### PR TITLE
Update Taxpasta update to reduce pipeline failures from overly strict profile validation 

### PR DESCRIPTION
**Pull request by jfy133**
As reported by multiple on slack and the taxpasta github repo

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,13 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [#484](https://github.com/nf-core/taxprofiler/pull/484) Improved input validation to immediately fail if run accession IDs within a given sample ID are not unique (❤️ to @sofstam for reporting, fixed by @jfy133)
 - [#491](https://github.com/nf-core/taxprofiler/pull/491) Added flag to publish intermediate bracken files (❤️ to @ewissel for reporting, fixed by @sofstam and @jfy133)
-- [489](https://github.com/nf-core/taxprofiler/pull/489) Fix KrakenUniq classified reads output format mismatch (❤️ to @SannaAb for reporting, fixed by @jfy133)
+- [#489](https://github.com/nf-core/taxprofiler/pull/489) Fix KrakenUniq classified reads output format mismatch (❤️ to @SannaAb for reporting, fixed by @jfy133)
+- [#493](https://github.com/nf-core/taxprofiler/pull/489) Stop TAXPASTA failures when profiles do not have exact compositionality (fixes by @Midnighter, @jfy133)
```
Comment (sofstam) [inline CHANGELOG.md:17]:
```suggestion
- [#495](https://github.com/nf-core/taxprofiler/pull/495) Stop TAXPASTA failures when profiles do not have exact compositionality (fixes by @Midnighter, @jfy133)
```

Comment (sofstam) [reply CHANGELOG.md:17]: https://github.com/nf-core/taxprofiler/pull/493

This should be added to the changelog.

File: conf/modules.config [hunk 2]
```diff
@@ -774,14 +773,15 @@ process {
...
+        ext.prefix = { "${meta.tool}_${meta.id}" }
         ext.args =  {
-            [
-                "-p ${meta.tool} -o ${meta.tool}_${meta.id}.${params.standardisation_taxpasta_format}",
-                params.taxpasta_add_name ?  "--add-name" : "",
-                params.taxpasta_add_rank ? "--add-rank" : "",
-                params.taxpasta_add_lineage ? "--add-lineage" : "",
-                params.taxpasta_add_idlineage ? "--add-id-lineage" : ""
-            ].join(' ').trim()
+                [
```
Comment (sofstam) [inline conf/modules.config:778]: I am somewhat confused here, don't we need to give the profiler argument?

Comment (jfy133) [reply conf/modules.config:778]: It has moved to the module invocation itself, supplied via a multimap!

**Review by sofstam — APPROVED** [review]

**Pull request merged**